src/main/java/com/aws/greengrass/device/ClientDevicesAuthService.java

```diff
@@ -231,6 +223,29 @@ protected void shutdown() throws InterruptedException {
         certificateManager.stopMonitors();
     }
 
+    @Override
+    public void postInject() {
+        super.postInject();
+        try {
+            authorizationHandler.registerComponent(this.getName(),
+                    new HashSet<>(Collections.singletonList(SUBSCRIBE_TO_CERTIFICATE_UPDATES)));
```

Comment:
although this is currently useless, go ahead and register all the other operations too, not just subscribe to cert updates.